src/server/document-controller.js

```diff
@@ -421,18 +331,6 @@ class DocumentController {
       (req, res) => this.handlePatchDocumentRestoreRevision(req, res)
     );
 
-    router.patch(
-      '/api/v1/docs/:documentId/archive',
-      [needsPermission(permissions.ARCHIVE_DOC), validateParams(documentIdParamsOrQuerySchema)],
-      (req, res) => this.handlePatchDocArchive(req, res)
-    );
-
-    router.patch(
-      '/api/v1/docs/:documentId/unarchive',
-      [needsPermission(permissions.ARCHIVE_DOC), validateParams(documentIdParamsOrQuerySchema)],
-      (req, res) => this.handlePatchDocUnarchive(req, res)
-    );
-
```

Comment:
Glad that you could ditch these 2 endpoints now.

Reply:
:+1: